console/web3ext/web3ext.go

```diff
@@ -706,43 +706,6 @@ const Miner_JS = `
 web3._extend({
```

Comment:
I think we can also remove the variable `Miner_JS`.

Reply:
Removed!